test/foundry/utils/helpers/StakingTestHelpers.t.sol

```diff
@@ -205,11 +205,6 @@ contract StakingTestHelpers is StakingV2Setup {
         stakingRewardsV2.unstakeEscrow(_amount);
     }
 
-    function unstakeEscrowSkipCooldownFundsV2(address _account, uint256 _amount) internal {
```

Comment:
**Thought:** Swapping this for a `unstakeEscrowAdmin` helper would allow quickly plugging back the tests I mentioned above.